src/test/java/spoon/test/template/testclasses/EnumAccessTemplate.java

```diff
@@ -13,15 +12,15 @@ public void method() throws Throwable {
 	}
 	
 	@Parameter("_AnEnum_")
-	CtTypeReference<?> anEnum;
+	String __AnEnum_;

```

Comment:
Yesterday I refactored my templates a lot and I have found a way how to write them in "refactoring friendly" way. I am using this:
```java
@Parameter(_AnEnum_.class.getSimpleName())
CtTypeReference<?> anEnum;
```
This PR actually kills this usefull approach.